styles/content.css

```diff
@@ -152,6 +152,14 @@
   @apply text-3xl;
 }
 
```

Comment:
This is another workaround of sorts. h3 & h4 section titles are getting flexbox treatment for when they need an E2E badge and works great. The FAQ page uses h2's but have other elements as well in the title (svg, potential badge, etc). 

The nuxt content (aka docs content) h2's are getting some vertical treatment to align the badge without flexbox. Targeting the div here isn't very specific but the badge class gets dynamic IDs appended to it making it tough to target.